src/cdk/menu/menu.md

```diff
@@ -117,45 +113,47 @@ your keyboard.
 
 ### Menu Items
 
-Both menu and menubar elements should exclusively contain menuitem elements. This directive allows
-the items to be navigated to via keyboard interaction.
+The `cdkMenuItem` directive allows the items to be navigated to via keyboard interaction.
```

Comment:
general comment: we should be consistent about "menu item" vs. "menuitem" throughout. I'd probably use "menu item" unless we're explicitly talking about the ARIA role